tests/Xamarin.Forms-Performance-Integration/Droid/Xamarin.Forms.Performance.Integration.Droid.csproj

```diff
@@ -40,9 +41,8 @@
     <WarningLevel>4</WarningLevel>
     <AndroidManagedSymbols>true</AndroidManagedSymbols>
     <AndroidUseSharedRuntime>false</AndroidUseSharedRuntime>
-    <AndroidSupportedAbis>armeabi-v7a;x86</AndroidSupportedAbis>
     <AndroidLinkTool Condition=" '$(AndroidLinkTool)' == '' ">r8</AndroidLinkTool>
-    <AndroidGenerateJniMarshalMethods>True</AndroidGenerateJniMarshalMethods>
+    <AndroidGenerateJniMarshalMethods Condition=" '$(HostOS)' != 'Linux' ">True</AndroidGenerateJniMarshalMethods>
```

Comment:
That is to avoid the mono crash, right?

Reply:
Yep, a stop-gap measure until we can figure out what's wrong.

Reply:
OK, I have opened an issue for that to not forget it. https://github.com/xamarin/xamarin-android/issues/2517